aiven/client/cliarg.py

```diff
@@ -2,15 +2,16 @@
 #
 # This file is under the Apache License, Version 2.0.
 # See the file `LICENSE` for details.
-
-from .argx import arg, UserError
+# type: ignore[attr-defined]
```

Comment:
Is it hard to avoid this ignore?

I guess it's mostly about `self.args`? Can we define `self.args` type as a protocol with `__setattribute__` like [here](https://github.com/aiven/aiven-core/blob/master/aiven/common/daemon.py#L25) for example?